lib/MusicBrainz/Server/Data/Recording.pm

```diff
@@ -422,9 +422,16 @@ sub find_recent_by_artists
 {
     my ($self, $artist_ids) = @_;
 
-    my $search_url = sprintf("http://%s/ws/2/recording/?query=title:\"\"",
-                              DBDefs->LUCENE_SERVER);
-
+    if (DBDefs->SEARCH_ENGINE == 'LUCENE')
+    {
```

Comment:
Can you fix the brace style to follow the majority of our codebase? :)

```
if (foo) {
    bar;
} else {
    baz;
}
```

Reply:
Sure, didn't know which one to use since I saw usages of both in Search.pm